third_party/packages/ansi_up/lib/ansi_up.dart

```diff
@@ -2,76 +2,306 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
-// ignore_for_file: avoid_classes_with_only_static_members
-// ignore_for_file: camel_case_types
-// ignore_for_file: non_constant_identifier_names
-// ignore_for_file: prefer_const_declarations
-
 /// ansi_up is an library that parses text containing ANSI color escape
 /// codes.
-@JS()
-library ansi_up;
-
-import 'package:js/js.dart';
 
-@JS()
 class AnsiUp {
-  external AnsiUp();
+  String version, _buffer;
```

Comment:
Make _buffer a `final StringBuffer` and put it on its own line.

Reply:
Why `StringBuffer` instead of `String` here? It's a little strange that this is called a buffer in the first place because there's only one spot where a string is added.

Reply:
sgtm. removing the append_buffer method completely sounds good.

Reply:
Did I do this part correctly? If I'm setting this `_text` field should it not be private with the underscore?